crates/typst-library/src/math/matrix.rs

```diff
@@ -110,6 +125,26 @@ pub struct MatElem {
     #[fold]
     pub augment: Option<Augment>,
 
+    /// The gap between rows.
+    ///
+    /// ```example
+    /// #set math.mat(row-gap: 3em)
+    /// $ mat(1, 2; 3, 4) $
+    /// ```
+    #[resolve]
+    #[fold]
+    pub row_gap: Option<Rel<Length>>,
+
+    /// The gap between columns.
+    ///
+    /// ```example
+    /// #set math.mat(col-gap: 3em)
```

Comment:
On a general note, 3em is pretty huge, so this will probably look a bit bad. Maybe pick a bit smaller value?